test/controllers/secure.register.filter.controller.spec.ts

```diff
@@ -48,7 +48,7 @@ describe( "SECURE REGISTER FILTER controller", () => {
 
       expect(resp.status).toEqual(200);
       expect(resp.text).toContain(SECURE_REGISTER_FILTER_PAGE_HEADING);
-      expect(resp.text).toContain(config.LANDING_URL);
+      expect(resp.text).toContain(SOLD_LAND_FILTER_URL);
```

Comment:
i think it should contain config.LANDING_PAGE_URL instead of SOLD_LAND_FILTER_URL

Reply:
It should be fine, the `SOLD_LAND_FILTER_URL` is related probably to the link present in the back button

Reply:
LANDING_PAGE_URL has ?start=0 on the end this will cause the test to fail.

Reply:
We can add the correct check of the header link only after the `ROE-1082`